src/epochs.lua

```diff
@@ -795,15 +796,17 @@ function epochs.pruneEpochs(timestamp)
 			Epochs[nextEpochIndex] = nil
 		else
 			local _, endTimestamp = epochs.getEpochTimestampsForIndex(nextEpochIndex)
-			if endTimestamp >= timestamp then
```

Comment:
This is a logical change, correct?

Reply:
reverted here - [23c1d55](https://github.com/ar-io/ar-io-network-process/pull/347/commits/23c1d553c2e474a6e76a6d4b1eed151ab33ee424)

i will write some more integration and unit tests validating these pruning behaviors in a separate PR before removing any logic